### PR TITLE
[Bugfix] Bound CallReadiness screens to the boundaries of the container

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What
Use ModalClone and a Layerhost instead of FluentUi provided Modal. 
image

# Why
Modal takes up full width and height of body and not the container it's called within.
https://skype.visualstudio.com/SPOOL/_workitems/edit/3086034
image

# How Tested
Locally on MacOS Safari

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by edwardlee-msft** [comment]
Note that the changes include an "align-items" prop which causes the width of the modal to be a bit thinner than before:
image

**Review by dmceachernmsft — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx [hunk 1]
```diff
@@ -82,6 +84,7 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   const [audioState, setAudioState] = useState<PermissionState | 'unsupported' | undefined>(undefined);
   /* @conditional-compile-remove(call-readiness) */
   getDevicePermissionState(setVideoState, setAudioState);
+  const modalLayerHostId = useId('callReadinessModalLayerhost');
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx:87]: We already have a composite-root level modal layer host id, can we use this instead so all modals, context menus etc. going forward all bind to the same root:
https://github.com/Azure/communication-ui-library/blob/d7d036e2d316fdc8081c3858b5a2a0bbf9db7b1b/packages/react-composites/src/composites/CallComposite/CallComposite.tsx#L377

Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx:87]: Good idea, I confirmed with FluentUi modal examples that multiple modals use 1 root modal layer host. 
I'll make the change now!

Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx]: Alright! Done, needed to play around with some transform in the modal but got it to work with the current layerhost. 
Also removed mobileView only for that specific layerhost as I found that it was not needed during my testing with the PIP modal. (but needs to be removed for call readiness modal to render on desktop using that layerhost).

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**